src/components/TopicsList/RenderTopicsList.js

```diff
@@ -53,9 +53,10 @@ const RenderTopicsList = props => {
   return (
     <>
       <div className="topics-list">
-        <Button onClick={displayTopicsICreated}>Topics I Created</Button>
-        <Button onClick={displayTopicsIJoined}>Topics I Joined</Button>
-        <h2>Topics</h2>
+        <div className="topic-buttons">
+          <Button onClick={displayTopicsICreated}>Topics I've Created</Button>
```

Comment:
This click (although I love its name) could actually be a single function. 

`onClick={(e) => displayTopics(e, data)}`

`const displayTopics = (e, data) => {
     // handle your prevent defaults
     // set data dynamically on state.
 }`